internal/cmd/info.go

```diff
@@ -75,7 +78,7 @@ func info() error {
 			groups = append(groups, g.Name)
 		}
 
-		fmt.Fprintln(w, "Groups:\t", strings.Join(groups, ", "))
+		fmt.Fprintf(w, "Groups:\t %v\n", groups)
```

Comment:
It is. I think it's appropriate for this context since there may be cases where the user is not a member of any group. In the previous case, it will output an empty string while with this change, it will output `[]` which IMO is clearer

Reply:
We could `if len(groups) > 0` to print this line, or something like this:

```go

type optionalGroups []api.Group

func (g optionalGroups) String() {
    if len(g)== 0 {
        return "none"
    }
    ... // format the slice
}

fmt.Fprintf(w, "Groups:\t%v\n", optionalGroups(userGroups))
...




```

Reply:
added `none` in parenthesis since `none` could, though unlikely, be a group name